en/techdocs/api/client/hoodie.account.md

```diff
@@ -157,18 +156,25 @@ synchronized to.
 | HoodieAccountUnconfirmedError  | Account has not been confirmed yet |
 | HoodieAccountNotFoundError     | Account could not be found |
 | HoodieError                    | _A custom error can be set by plugins, e.g. the account could be blocked due to missing payments_ |
-| HoodieConnectionError | Could not connect to Server                 |
+| HoodieConnectionError | Could not connect to server                 |
+
+The **signIn** method tries to sign in the user to an existing account.
 
-<br />
+#### NOTES
 
+All Hoodie data that existed in the browser's local store before signin will be cleared on signin. To prevent data loss, you can pass **{moveData: true}** as the options argument, this will move current data (created anonymously or by another account) to the account the user signs in to.
```

Comment:
Good catch. It *does* work, but Hoodie fails to clear the local store before doing the second sign in. So when you sign in to account B while signed into A, you'll get both A and B's todos. If you sign back into A, you'll get A's todos twice, and B's todos once. Sign back into B, you'll have both twice. 

So this should definitely be handled differently, but I'll remove it from the docs for now.